src/treeChooser/treeChooser.controller.js

```diff
@@ -480,6 +483,12 @@ function TreeChooserController(
       var checkItem = vm.itemsIndex[id];
       if (checkItem) {
         checkItem.setSelected(true);
+        if (vm.isInitialLoad) {
```

Comment:
Why is there special logic for initialLoad?

Reply:
I need at initial initialization of a component when in it already there is data once to check up and expose tabindex on necessity. I looked at all functions. Decided that this is the best option. I'll be glad if you offer a more elegant solution

Reply:
This should probably go in the syncModelToItems. If the underlying model changes without tree-chooser interaction, this will sync them

Reply:
Are you going to make this change?

Reply:
I'm sorry, i don't understand what do you mean.
This block already in syncModelToItems. or do you mean that the block needs to be taken out of the cycle? This isn't a good idea, because need checkItem condition.